src/modules/appData/services/index.ts

```diff
@@ -0,0 +1,9 @@
+export type UploadAppDataDoc = (appDataDoc: string) => Promise<string>
+
+export const uploadAppDataDocOrderbookApi: UploadAppDataDoc = async (appDataDoc) => {
+  // TODO: Implement logic to post the doc to the orderbook API
+  // https://cowservices.slack.com/archives/C0375NV72SC/p1690300109648539?thread_ts=1690297997.334099&cid=C0375NV72SC
+  console.log('POST api.cow.fi/v1/app_datas, with content', appDataDoc)
```

Comment:
placeholder to add the call to the SDK (which needs to be updated with the latest swagger)